### PR TITLE
RATIS-1285. Fix the wrong links from Ratis website

### PR DESCRIPTION
**Pull request by runzhiwang**
## What changes were proposed in this pull request?

There are wrong link in ratis website. 

> Getting started
> 
> Ratis is a Raft protocol library in Java. It’s not a standalone server application like Zookeeper or Consul.
> 

Only need to rebuild, because asf-site-soure has fix this: https://github.com/apache/incubator-ratis/commit/57725bb419ceaa8fc08fdf20b29546b8d4614273, but did not commit asf-site.

### CODE REVIEW

**Comment by runzhiwang** [comment]
@szetszwo Could you help review this ?

File: index.html [hunk 1]
```diff
@@ -283,7 +283,7 @@ <h2>Download</h2>
             <h2>Getting started</h2>
             <!-- raw HTML omitted -->
 <!-- raw HTML omitted -->
-<p>Ratis is a <a href="https://raft.github.io/%22">Raft</a> protocol <em>library</em> in Java. It&rsquo;s not a standalone server application like Zookeeper or Consul.</p>
+<p>Ratis is a <a href="https://raft.github.io/">Raft</a> protocol <em>library</em> in Java. It&rsquo;s not a standalone server application like Zookeeper or Consul.</p>
```
Comment (runzhiwang) [inline index.html:286]: This is the real fix.

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Comment by runzhiwang** [comment]
@szetszwo Thanks for review. I have merged it.

**Comment by amaliujia** [comment]
+1 thanks!

**Pull request merged**